tests/karma/unit/services/delete-doc.js

```diff
@@ -76,22 +104,21 @@ describe('DeleteDoc service', function() {
         _id: 'b'
       }
     };
-    get
-      .onFirstCall().returns(KarmaUtils.mockPromise(null, person))
-      .onSecondCall().returns(KarmaUtils.mockPromise(null, clinic));
-    put.returns(KarmaUtils.mockPromise());
-    service('a', function(err, actual) {
-      chai.expect(get.calledTwice).to.equal(true);
-      chai.expect(put.calledTwice).to.equal(true);
-      chai.expect(get.firstCall.args[0]).to.equal('a');
-      chai.expect(get.secondCall.args[0]).to.equal('b');
-      chai.expect(put.firstCall.args[0].contact).to.equal(null);
-      chai.expect(actual._deleted).to.equal(true);
-      done();
+    get.returns(KarmaUtils.mockPromise(null, clinic));
+    bulkDocs.returns(KarmaUtils.mockPromise());
+    return service(person).then(function() {
+      chai.expect(get.callCount).to.equal(1);
+      chai.expect(get.args[0][0]).to.equal(clinic._id);
+      chai.expect(bulkDocs.callCount).to.equal(1);
+      chai.expect(bulkDocs.args[0][0].length).to.equal(2);
+      chai.expect(bulkDocs.args[0][0][0]._id).to.equal(person._id);
+      chai.expect(bulkDocs.args[0][0][0]._deleted).to.equal(true);
+      chai.expect(bulkDocs.args[0][0][1]._id).to.equal(clinic._id);
+      chai.expect(bulkDocs.args[0][0][1].contact).to.equal(null);
     });
   });
 
-  it('done update clinic when phone does not match', function(done) {
+  it('does not update clinic when phone does not match', function() {
```

Comment:
Sorry, I wasn't clear. 
I mean, why do we not update the clinic when the phone doesn't match? (I could maybe leave such questions out of the code review, which has enough comments as it is :D )